.vib/harbor-core/goss/harbor-core.yaml

```diff
@@ -0,0 +1,35 @@
+group:      
+  harbor:          
+    exists: true  
+user:      
+  harbor:          
+    exists: true
+file:
+  /etc/core:        
+    exists: true          
+    mode: "0775"          
+    owner: harbor          
+    filetype: directory
+  /data:
+    exists: true
+    mode: "0775"
+    owner: harbor
+    filetype: directory
+  /etc/core/certificates:
+    exists: true
+    filetype: symlink
+  /etc/core/ca_download:
+    exists: true
+    filetype: symlink
+  /etc/core/token:
+    exists: true
+    filetype: symlink
```

Comment:
We should add the `linked-to` option for these symlinks. Ref: [Goss' doc](https://github.com/goss-org/goss/blob/master/docs/manual.md#file)